src/table/components/__tests__/PaginationContent.spec.tsx

```diff
@@ -36,13 +52,17 @@ describe('<PaginationContent />', () => {
         rect={rect}
         handleChangePage={handleChangePage}
         announce={announce}
+        selectionsAPI={SelectionsAPI}
+        rootElement={rootElement}
+        layout={layout}
+        changeSortOrder={changeSortOrder}
```

Comment:
These are needed since they are on RootProps, that we extend. Could be omitted but not sure it is worth it. In production these props are actually passed since we are spreading props from tableWrapper

Reply:
@haxxmaxx What if you use Pick<T, K> from RootProps instead ?

Reply:
yeah I don't know, none of the solutions are good IMO. I'm thinking we might have a type called `commonProps` or something. That one has the props that all components have, then you extend it with props needed for each component. The common props is completely arbitrary, it is just the overlap of all components. For now that is the best solution I can think of.